.github/workflows/helm-release.yaml

```diff
@@ -33,7 +33,10 @@ jobs:
           echo "RELEASE_VERSION=$(date +'%Y.%-m.%-d%H%M%S')" >> $GITHUB_ENV
           echo "PREFECT_VERSION=$(\
             git ls-remote --tags --refs --sort="v:refname" \
-            https://github.com/PrefectHQ/prefect.git '[3].*.[!rc]' | tail -n1 | sed 's/.*\///' \
+              https://github.com/PrefectHQ/prefect.git '3.*.*' \
+              | grep -E -v 'rc|dev' \
```

Comment:
`-E` allows extended syntax, allowing us to use `|` as an OR statement rather than having to use multiple `grep -v` statements

Reply:
nice!

Reply:
LGTM
```
➜ git ls-remote --tags --refs --sort="v:refname" \
➜               https://github.com/PrefectHQ/prefect.git '3.*.*' \
➜               | grep -E -v 'rc|dev' \
➜               | tail -n1 \
➜               | sed 's/.*\///' \
➜
3.1.12
```